src/main/java/easytests/api/v1/controllers/QuestionsController.java

```diff
@@ -37,8 +40,28 @@
     /**
      * show(questionId)
      */
-    /**
-     * delete(questionId)
-     */
+    @DeleteMapping("/{questionId}")
+    public void delete(@PathVariable Integer questionId) throws NotFoundException, ForbiddenException {
+        final QuestionModelInterface questionModel = this.getQuestionModel(questionId);
+
+        if (!this.acl.hasAccess(questionModel)) {
+            throw new ForbiddenException();
+        }
+        
+        this.questionsService.delete(questionModel);
```

Comment:
nope, that command only delete question. without answers, and deletion will fail on FK constraint.